editoast/src/views/timetable/stdcm.rs

```diff
@@ -251,7 +255,38 @@ async fn stdcm(
 
     let stdcm_response = stdcm_request.fetch(core_client.as_ref()).await?;
 
-    // 6. Handle STDCM Core Response
+    // 6. Check if the current tracing level is debug or lower, and if so, log STDCM request and response
+    if tracing::level_filters::LevelFilter::current() >= tracing::Level::DEBUG {
```

Comment:
We have to do the following `clone` outside of `spawn`, otherwise, we get a borrowing error.
```rust
let stdcm_response_for_spawn = stdcm_response.clone();
```

Reply:
This has been resolved with #9947.